impl_core/src/test/java/io/opencensus/implcore/tags/ScopedTagMapTest.java

```diff
@@ -109,4 +119,35 @@ public void addToCurrentTagsWithBuilder() {
       scope1.close();
     }
   }
+
+  @Test
+  public void multiScopeTagMapWithMetadata() {
```

Comment:
This test matches https://github.com/census-instrumentation/opencensus-specs/pull/233/files#diff-7ee77560805f6772a65a8add67f0d692R95.